src/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, ReactNode } from 'react';
+import { K } from './types';
+import classes from './Card.module.scss';
+
+const Card: FC<{data: K}> = ({data}) => {
+  return (
+    data.map((category)=> {
+      const categoryTitle = Object.keys(category)[0];
+      const items = category[categoryTitle];
+      const categoryItems = items.map((item) => 
```

Comment:
Why is it map inside of map? Ideally this component should acept one piece of data with title, optional description and dishes list but not the whole bunch of data.